build.gradle

```diff
@@ -20,7 +20,9 @@ repositories {
         url = uri('https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot')
     }
 }
-
+ext {
+    springBootVersion = '2.6.2'
```

Comment:
Make sure that this version is same as other spring dependencies

Reply:
Yes